resources/migrations/20200914093309-add-tasks.up.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE IF NOT EXISTS tasks(
```

Comment:
We've decided to avoid using `IF NOT EXISTS`, because we would only do up migrations in prod. The down migrations are just for developer convenience. Up migrations should fail if tables exist.